### PR TITLE
[Various] Migrate more images

### PR DESCRIPTION
**Pull request by kodster28**
Continuation of PCX-6568

### CODE REVIEW

File: content/fundamentals/account-and-billing/account-maintenance/delete-account.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Deleting your account is permanent. Any accounts where you are the primary owner
 
 3. Select **Delete this user**
```
Comment (pedrosousa) [inline content/fundamentals/account-and-billing/account-maintenance/delete-account.md]:
```suggestion
3. Select **Delete this user**.
```

File: content/fundamentals/data-products/analytics-integrations/elastic/index.md [hunk 2]
```diff
@@ -269,13 +269,13 @@ If you run into any other issues, take note of the exact return message and cont
 To analyze the health status of the Lambda function from Kibana:
 
 1.  From Elastic Cloud, launch the Kibana console.
-    ![Lauching Elastic Cloud Kibana console](/fundamentals/static/images/elastic/screenshots/elastic-cloud-console.png)
+    ![Lauching Elastic Cloud Kibana console](/images/fundamentals/elastic/screenshots/elastic-cloud-console.png)
```
Comment (pedrosousa) [inline content/fundamentals/data-products/analytics-integrations/elastic/index.md]:
```suggestion
    ![Launching Elastic Cloud Kibana console](/images/fundamentals/elastic/screenshots/elastic-cloud-console.png)
```

File: content/fundamentals/get-started/basic-tasks/find-account-and-zone-ids.md [hunk 3]
```diff
@@ -13,10 +13,10 @@ To find your zone and account IDs:
 1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com/login) and select your account and domain.
 2. On the **Overview** page (the landing page for your domain), find the **API** section.
 
-    ![Screenshot of the Overview page with the API section highlighted](/fundamentals/static/images/get-started/dash-overview-api-highlighted.png)
+    ![Screenshot of the Overview page with the API section highlighted](/images/fundamentals/get-started/dash-overview-api-highlighted.png)
 
 3. The **API** section contains your **Zone ID** and **Account ID**. To copy these values for API commands or other tasks, click **Click to copy**.
```
Comment (pedrosousa) [inline content/fundamentals/get-started/basic-tasks/find-account-and-zone-ids.md]:
```suggestion
3. The **API** section contains your **Zone ID** and **Account ID**. To copy these values for API commands or other tasks, select **Click to copy**.
```

File: content/cloudflare-for-platforms/cloudflare-for-saas/_index.md
Comment (pedrosousa) [inline content/cloudflare-for-platforms/cloudflare-for-saas/_index.md:1]: Should we also move images kept near the corresponding section, like `content/cloudflare-for-platforms/workers-for-platforms/learning/media/wfp.png`, being used here?
https://github.com/cloudflare/cloudflare-docs/blob/production/content/cloudflare-for-platforms/workers-for-platforms/learning/how-workers-for-platforms-works.md?plain=1#L51

File: content/fundamentals/account-and-billing/account-maintenance/change-plan.md
Comment (pedrosousa) [inline content/fundamentals/account-and-billing/account-maintenance/change-plan.md:1]: Nit: Not sure if you're approaching this by tile, but there are still a few images in Fundamentals that were not moved (used in Markdown files under "/fundamentals/api/").

(Also in "What is Cloudflare?", even though it's an SVG.)

**Review by pedrosousa — APPROVED** [review]

File: content/cloudflare-for-platforms/cloudflare-for-saas/_index.md
Comment (kodster28) [reply content/cloudflare-for-platforms/cloudflare-for-saas/_index.md:1]: Ohh, I missed that one, good catch.

File: content/fundamentals/account-and-billing/account-maintenance/change-plan.md
Comment (kodster28) [reply content/fundamentals/account-and-billing/account-maintenance/change-plan.md:1]: Ohh, I missed those ones, good catch.

For the `What is Cloudflare?`, I haven't figured out how to support actually rendering SVG using the `/assets` folder, so figured I'd exclude for now.

**Pull request merged**